{{ cookiecutter.format }}/app/src/main/python-briefcase/rubicon/java/android_events.py

```diff
@@ -125,12 +125,6 @@ def enqueue_android_wakeup_for_delayed_tasks(self):
             self.run_delayed_tasks, timeout * 1000,
         )
 
-    def _set_coroutine_origin_tracking(self, debug):
-        # If running on Python 3.7 or 3.8, integrate with upstream event loop's debug feature, allowing
-        # unawaited coroutines to have some useful info logged. See https://bugs.python.org/issue32591
-        if hasattr(super(), "_set_coroutine_origin_tracking"):
-            super()._set_coroutine_origin_tracking(debug)
-
```

Comment:
Based on the comment, this seems unnecessary now; is it safe to remove?

Reply:
Given the age of that link, another possible interpretation of the comment is that this feature was added in Python 3.7 and has never been removed. However, this call may still be unnecessary for other reasons, or maybe it's been replaced with a public API. Needs more investigation.

Reply:
But as @freakboy3742 says below, the whole `python-briefcase` directory can now be removed. This means that the entire `sourceSets` block in build.gradle can also be removed, because `python.srcDirs = ["src/main/python"]` is already the default setting.